src/ids.rs

```diff
@@ -1127,13 +1119,11 @@ memory[fp] = ids.ok_ref
 
             let py_result = py.run(code, Some(globals), None);
 
-            assert_eq!(
-                py_result.map_err(|err| to_vm_error(err, py)),
-                Err(to_vm_error(
-                    PyValueError::new_err(VirtualMachineError::NoRegisterInReference.to_string()),
-                    py
-                ))
-            );
+            assert!(py_result.unwrap_err().to_string().contains(
+                &VirtualMachineError::NoRegisterInReference
+                    .to_string()
+                    .to_string()
```

Comment:
Why do you need to `.to_string()` twice?

Reply:
Just a copy paste error.
Removed [a0fe931](https://github.com/lambdaclass/cairo-rs-py/pull/175/commits/a0fe93135a090b3f0e7351f0ba4d1bf5150a19eb)

Reply:
https://github.com/lambdaclass/cairo-rs-py/issues/180